appx/sconscript

```diff
@@ -63,7 +63,8 @@ signExec=env['signExec'] if env['certFile'] else None
 # Files from NVDA's distribution that cannot be included in the appx due to policy or security restrictions
 excludedDistFiles=[
 	'nvda_eoaProxy.exe',
-	'nvda_service.exe',
+	# TODO should this be excluded?
+	# 'nvda_dmp.exe',

```

Comment:
I don't think this should be excluded. Though perhaps a second exe may need to be specifically mentioned in the manifest... We don't really use this code at the momement, and would only really know once we build one again and verify it with Microsoft tools. But it is correct that nvda_service can be removed as we don't create it anymore.